week3/hw-week3.playground/Contents.swift

```diff
@@ -0,0 +1,163 @@
+import UIKit
+
+//a.Create an array of Int called nums with values 0 through 20. Iterate over the Array and print the even numbers.
+
+print("**************")
+print("a")
+print("**************")
+
+var nums = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
+
+for i in nums{
+    if i%2 == 0 {
+        print(i)
+    }
+}
+
+/*b) In your Playground, define the following String:
+let sentence = “The qUIck bRown fOx jumpEd over the lAzy doG”
+
+Iterate over sentence counting the vowels (a, e, i, o, u), ignoring the case.*/
+
+print("**************")
+print("b")
+print("**************")
+
+let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"
+let vowels = ["a","e","i","o","u"]
+var count=0
+
+let char = Array(sentence.lowercased())
+
+for str in char{
+    if(vowels.contains(String(str))) {
+        count=count+1
+    }
+}
+
+print("The number of vowel is \(count)")
+
+
+/*
+ 
+ c) Create two arrays of Int with the values of 0 through 4. Use a nested for loop to print a mini multiplication table. The output, which should be multiple lines, should be in the following format:
+ 0 * 0 = 0
+
+ The format follows the number from the first array, space, followed by the character *, space,  followed by the number from the second array, space,  followed by =, space, followed by the result of the number from the first array multiplied by the number from the second array (just like the example above). There are a couple of ways to achieve this, but String Interpolation is the easiest.
+ 
+ */
+
+print("**************")
+print("c")
+print("**************")
+
+let a1 = [0,1,2,3,4]
```

Comment:
Please name your variables and constants better, a1 and a2 doesn't really convey any meaning.